tests/test_urdf.py

```diff
@@ -0,0 +1,39 @@
+
+import math
+import os
+
+import pytest
+import xacro
+from urdf_parser_py.urdf import URDF
+
+model_data = [
+    ("med7", 25.5),
+    ("med14", 32.3),
+    ("iiwa7", 23.9),
+    ("iiwa14", 29.9)
+]
+
+@pytest.mark.parametrize("model, mass", model_data)
+def test_urdf_mass(model: str, mass: float, abs_tol: float=1.e-5) -> None:
+    r"""Tests the total mass withtin the URDF files against
+    the KUKA reference mass.
+
+    Args:
+        model (str): Model name
+        mass (float): Reference mass
+        abs_tol (float): Absolute tolerance in kg, 1.e-5 kg = 0.01 g
+    """
+    path = f"{os.getcwd()}/src/lbr_fri_ros2_stack/lbr_description/urdf/{model}/{model}.urdf.xacro"
```

Comment:
Not ideal to get path using `os.getcwd`, better to set the reletive path inside `lbr_description` and get the absolute using [`get_package_share_directory`](https://answers.ros.org/question/288501/ros2-equivalent-of-rospackagegetpath/).

Also, Python3 has [`pathlib`](https://docs.python.org/3/library/pathlib.html) - this will be safer to use since ROS2 supports Windows.

Reply:
update path via `get_package_share_directory`. `os.join` is also os independent